pandas/core/dtypes/cast.py

```diff
@@ -1083,6 +1083,7 @@ def convert_dtypes(
     convert_integer: bool = True,
     convert_boolean: bool = True,
     convert_floating: bool = True,
+    infer_objects: bool = False,
```

Comment:
Nit: Could you add to the docstring below?

Reply:
added